### PR TITLE
Refine CLI flag descriptions for clarity

### PR DESCRIPTION
**Pull request by jacobthesheep**
## Changes

- Updated wording of `create-astro` CLI flags

## Testing

N/A

## Docs

@withastro/maintainers-docs should probably add docs for this

### CODE REVIEW

**Review by natemoo-re — APPROVED** [review]
Thank you so much for these improvements! I know people will be thrilled with them.

**Comment by natemoo-re** [comment]
Waiting for a review from @withastro/maintainers-docs which I've been told is coming tomorrow!

**Comment by sarah11918** [comment]
Hi @jacobthesheep ! Can you please revise this PR to only include the new options you're adding, and no changes to existing flags? Thanks!

**Review by sarah11918 — DISMISSED** [review]
Requesting that only new flags be added; removing changes to existing content.

**Comment by jacobthesheep** [comment]
> Requesting that only new flags be added; removing changes to existing content.

I'm not adding new flags. I just updated the wording for existing flags.

File: packages/create-astro/README.md [hunk 1]
```diff
@@ -45,16 +45,19 @@ npm create astro@latest my-astro-project -- --template cassidoo/shopify-react-as
...
+| `--template <name>`        | Specify a template.                                        |
+| `--install / --no-install` | Toggle dependency installation.                            |
+| `--git / --no-git`         | Toggle git repository initialization.                      |
+| `--yes (-y)`               | Accept all default prompts.                                |
+| `--no (-n)`                | Decline all default prompts.                               |
+| `--dry-run`                | Simulate steps without actual execution.                   |
+| `--skip-houston`           | Skip the Houston animation.                                |
+| `--ref`                    | Specify an Astro branch (default: latest).                 |
+| `--fancy`                  | Enable full Unicode support for Windows.                   |
+| `--typescript <option>`    | Choose a TypeScript option: `strict`, `strictest`, `relaxed`. |
```
Comment (ematipico) [inline packages/create-astro/README.md:60]: It seems these were added?

Comment (sarah11918) [reply packages/create-astro/README.md]: Yes, but they weren't in the text before. I would like only the new items to the text in this PR, please!

File: packages/create-astro/src/actions/help.ts [hunk 2]
```diff
@@ -11,12 +11,12 @@ export function help() {
...
 				['--git / --no-git', 'Initialize git repo (or not).'],
-				['--yes (-y)', 'Skip all prompt by accepting defaults.'],
-				['--no (-n)', 'Skip all prompt by declining defaults.'],
+				['--yes (-y)', 'Skip all prompts by accepting defaults.'],
+				['--no (-n)', 'Skip all prompts by declining defaults.'],
 				['--dry-run', 'Walk through steps without executing.'],
 				['--skip-houston', 'Skip Houston animation.'],
-				['--ref', 'Choose astro branch (default: latest).'],
-				['--fancy', 'Enable full unicode support for Windows.'],
+				['--ref', 'Choose astro Astro branch (default: latest).'],
```
Comment (sarah11918) [inline packages/create-astro/src/actions/help.ts:18]:
```suggestion
				['--ref', 'Choose astro branch (default: latest).'],
```
 duplicated word. (And, we would typically use lowercase to refer to the repository, so I think this is the one we'd keep.)

File: packages/create-astro/README.md [hunk 3]
```diff
@@ -45,16 +45,19 @@ npm create astro@latest my-astro-project -- --template cassidoo/shopify-react-as
...
-| `--skip-houston`             | Skip Houston animation.                                |
-| `--typescript <option>`      | TypeScript option: `strict` / `strictest` / `relaxed`. |
+| Name                       | Description                                                |
+| :------------------------- | :--------------------------------------------------------- |
+| `--help (-h)`              | Display available flags.                                   |
+| `--template <name>`        | Specify your template.                                     |
+| `--install / --no-install` | Install dependencies (or not).                             |
+| `--git / --no-git`         | Initialize git repo (or not).                              |
+| `--yes (-y)`               | Skip all prompts by accepting defaults.                    |
+| `--no (-n)`                | Skip all prompts by declining defaults.                    |
```
Comment (sarah11918) [inline packages/create-astro/README.md:55]:
```suggestion
| `--install` / `--no-install` | Install dependencies (or not).                             |
| `--git` / `--no-git`         | Initialize git repo (or not).                              |
| `--yes`  (`-y`)              | Skip all prompts by accepting defaults.                    |
| `--no`  (`-n`)               | Skip all prompts by declining defaults.                    |
```
I'm confused by the formatting change here. As these are two separate options, shouldn't they be indicated separately as they were before? Was this wrong before?

File: packages/create-astro/README.md [hunk 4]
```diff
@@ -45,16 +45,19 @@ npm create astro@latest my-astro-project -- --template cassidoo/shopify-react-as
...
-| `--install` / `--no-install` | Install dependencies (or not).                         |
-| `--git` / `--no-git`         | Initialize git repo (or not).                          |
-| `--yes` (`-y`)               | Skip all prompt by accepting defaults.                 |
-| `--no` (`-n`)                | Skip all prompt by declining defaults.                 |
-| `--dry-run`                  | Walk through steps without executing.                  |
-| `--skip-houston`             | Skip Houston animation.                                |
-| `--typescript <option>`      | TypeScript option: `strict` / `strictest` / `relaxed`. |
+| Name                       | Description                                                |
+| :------------------------- | :--------------------------------------------------------- |
+| `--help (-h)`              | Display available flags.                                   |
```
Comment (sarah11918) [inline packages/create-astro/README.md:50]:
```suggestion
| `--help`  (`-h`)             | Display available flags.                                   |
```
Just on the assumption this should be like the others!

**Review by sarah11918 — COMMENTED** [review]
Thanks for catching these typos and adding some missing flags, @jacobthesheep !

I'm not sure about the formatting changes though, that doesn't seem right to me, so let me know if I'm missing something.

**Comment by jacobthesheep** [comment]
> Thanks for catching these typos and adding some missing flags, @jacobthesheep !
> 
> I'm not sure about the formatting changes though, that doesn't seem right to me, so let me know if I'm missing something.

Sorry about that. Thanks for catching those.

**Review by natemoo-re — APPROVED** [review]
Thanks for adding the missing info and the typo fixes! Appreciate it.

**Pull request merged**